hazelcast/src/main/java/com/hazelcast/client/console/SqlConsole.java

```diff
@@ -187,13 +192,18 @@ private static void executeSqlCmd(
             int[] colWidths = determineColumnWidths(rowMetadata);
             Alignment[] alignments = determineAlignments(rowMetadata);
 
-            // this is a result with rows. Print the header and rows, watch for concurrent cancellation
-            printMetadataInfo(rowMetadata, colWidths, alignments, out);
-
+            boolean isExplainQuery = startsWithIgnoreCase(command, "explain");
```

Comment:
We should do the same thing as here: https://github.com/hazelcast/hazelcast/blob/021e2af77ad6ba6d32d56a623fbe43f30fd76864/hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientResult.java#L88

That is, we should also ignore the leading whitespace. We don't need to extract it to a common StringUtils, it's too specific IMO. Or we can extract it to some SQL utility class and name it `startsWithKeyword`. And instead of converting the prefix to lower case we can assert that.

Reply:
We can perhaps add an assert if this is the case.